skyportal/handlers/api/photometry_validation.py

```diff
@@ -0,0 +1,323 @@
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import permissions
+from ..base import BaseHandler
+
+from ...models import Photometry, PhotometryValidation
+
+
+class Validator(Schema):
+    method = fields.Str(required=True)
+    photometry_id = fields.Integer()
+    validated = fields.Boolean(
+        truthy=['true', 'True', 'validated', True],
+        falsy=['false', 'False', 'rejected', False],
+        required=False,
+    )
+    explanation = fields.String(required=False)
+    notes = fields.String(required=False)
+
+    @validates_schema
+    def validate_requires(self, data, **kwargs):
+        if 'method' not in data:
+            raise ValidationError('method is required')
+        if data['method'] not in ['POST', 'PATCH', 'DELETE']:
+            raise ValidationError('method must be one of POST, PATCH or DELETE')
+        if (
+            data['method'] == 'PATCH'
+            or data['method'] == 'DELETE'
+            or data['method'] == 'POST'
+        ):
+            if 'photometry_id' not in data:
+                raise ValidationError('Missing required fields')
+            if data['photometry_id'] is None:
+                raise ValidationError('Missing required fields')
+
+
+class PhotometryValidationHandler(BaseHandler):
+    @permissions(['Manage sources'])
+    async def post(self, photometry_id):
+        """
+        ---
+        description: Validate or reject a photometric point
+        tags:
+          - photometryvalidations
+        parameters:
+          - in: path
+            name: photometry_id
+            required: true
+            schema:
+              type: int
+            description: Photometry ID
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  validated:
+                    type: boolean
+                    description: Whether the source is validated (True) or rejected (False)
+                required:
+                  - validated
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Success'
+                    - type: object
+                      properties:
+                        data:
+                          type: object
+                          properties:
+                            id:
+                              type: int
+                              description: The id of the photomety_validation
+          400:
+            content:
+              application/json:
+                schema: Error
+
+        """
+        data = self.get_json()
+
+        validated = data.get('validated')
+        explanation = data.get('explanation')
+        notes = data.get('notes')
+
+        validator_instance = Validator()
+        params_to_be_validated = {
+            'method': 'POST',
```

Comment:
Double checking: should `method` be set to `POST` by default?

Reply:
For this part of the handler, yes.